python/py-enrich/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-enrich
+version             1.2.6
+revision            0
+python.versions     36 37 38 39
+
+platforms           darwin
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         Enrich adds few missing features to the wonderful rich library.
+long_description    Enriched extends rich library functionality with a set of \
+                    changes that were not accepted to rich itself.
+homepage            https://github.com/pycontribs/enrich
+
+checksums           rmd160  e695a61001c51770df82a932d277534cafd69f8d \
+                    sha256  0e99ff57d87f7b5def0ca79917e88fb9351aa0d52e228ee38bff7cd858315fe4 \
+                    size    15581
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-pip \
+                    port:py${python.version}-setuptools_scm_git_archive \
+                    port:py${python.version}-toml \
+                    port:py${python.version}-wheel
+
+    depends_lib-append \
+                    port:py${python.version}-rich
```

Comment:
you will need to add py36/py37 subports to `py-rich` for this to work (see also the `lint` messages)